adminSiteClient/GdocsDateline.tsx

```diff
@@ -32,9 +32,10 @@ export const GdocsPublishedAt = <T extends OwidGdoc>({
             <DatePicker
                 onChange={onChangePublishedAt}
                 value={publishedAt ? dayjs(publishedAt) : undefined}
-                format="ddd, MMM D, YYYY"
+                format="ddd, MMM D, YYYY HH:mm"
```

Comment:
I think we should render this as UTC instead of the user's local timezone (with a tooltip/label that explains this)

That way, the authors can all use the same time for "publishes in the morning" wherever they are in the world

Reply:
Yeah I see your point that authors might get the conversion wrong and expect to have an article be published immediately, but instead be off-by-one.

But I think as long as we have the "Publish Now" button that should be okay? If we want to be super sure, we could also add a button that sets the DatePicker to "now" (with a fix for the seconds caveat described in the comment just below this code change)

Reply:
Ed likes the idea of using UTC, but I wasn't able to make it work, see https://github.com/ant-design/ant-design/issues/45880

Since we want to ship this PR this week, I propose we won't block on this and if we really want to, we can switch to a different date picker or something later.

Reply:
That's not the best related issue at ant-design, but I tried several ways of making it work and failed.